app/views/RoomsListView/ServerDropdown.js

```diff
@@ -202,43 +202,8 @@ class ServerDropdown extends Component {
 		const { server, theme } = this.props;
 
 		return (
-			<Pressable
-				onPress={() => this.select(item.id)}
-				onLongPress={() => (item.id === server || this.remove(item.id))}
-				testID={`rooms-list-header-server-${ item.id }`}
-				android_ripple={{
-					color: themes[theme].bannerBackground
-				}}
-				style={({ pressed }) => ({
-					backgroundColor: isIOS && pressed
-						? themes[theme].bannerBackground
-						: 'transparent'
-				})}
-			>
-				<View style={styles.serverItemContainer}>
-					{item.iconURL
-						? (
-							<Image
-								source={{ uri: item.iconURL }}
-								defaultSource={require('../../static/images/logo.png')}
-								style={styles.serverIcon}
-								onError={() => console.warn('error loading serverIcon')}
-							/>
-						)
-						: (
-							<Image
-								source={require('../../static/images/logo.png')}
-								style={styles.serverIcon}
-							/>
-						)
-					}
-					<View style={styles.serverTextContainer}>
-						<Text style={[styles.serverName, { color: themes[theme].titleText }]} numberOfLines={1}>{item.name || item.id}</Text>
-						<Text style={[styles.serverUrl, { color: themes[theme].auxiliaryText }]} numberOfLines={1}>{item.id}</Text>
-					</View>
-					{item.id === server ? <Check theme={theme} /> : null}
-				</View>
-			</Pressable>
+			<ServerItem item={item} onPress={() => this.select(item.id)} onLongPress={() => (item.id === server || this.remove(item.id))} hasCheck={item.id === server} theme={theme} />
```

Comment:
We don't have Prettier yet and our current linter let you do it like this, but you should be indenting some parts of the code just like Prettier would.
In this case, one prop per line is way better.